src/main/webapp/scripts/toggle.js

```diff
@@ -50,7 +50,7 @@ function renderTemplateObj(template, objs, keywords) {
     let term = keywords[i];
     let keywordsObj = new Object();
     keywordsObj.term = term;
-    keywordsObj.news = objs[0].articles[term];
+    keywordsObj.news = objs[0].books[term];
```

Comment:
[Nit] Change .news for .books and update the mustache template accordingly.

Reply:
Good call -- I also changed anything with the word 'news' in it to 'books' in the html div's and main.css file.